components/Navigation/styleNavigation.css

```diff
@@ -0,0 +1,51 @@
+@import url(../../css/main.css);
+
+.navbar {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	color: var(--primaryColor);
+}
+
+.container {
+	display: flex;
+}
+
+.sidebar {
+	display: flex;
+	flex-direction: column;
+	width: 250px;
+	padding-left: 2.5rem;
+}
+.main {
+	display: flex;
+	flex-grow: 1;
+	flex-direction: column;
+}
+
+.textContainer {
+	/* display: flex; */
+	justify-content: space-evenly;
+	align-items: center;
+	background-color: var(--lightGrey);
+	width: 95%;
+	margin: 1rem;
+}
+
+
+li {
+	list-style: none;
+	font-size: 0.2rem;
+	color: white;
+}
+h2 {
```

Comment:
It would be better if you create a class name for h2 etc. because this might cause problems later.